setup.py

```diff
@@ -13,6 +13,7 @@
     'numpy==1.15.4',
     'pandas==0.23.4',
     'scipy==1.1.0',
+    'Faker==1.0.1',
```

Comment:
Please use open dependency specification: `>=`
Also do that for the other dependencies. There is no need to hard fix all the versions.